objects/endpoint.rb

```diff
@@ -79,7 +79,9 @@ def to_h
       log: @item['log']
     }
     h[:updated] = Time.at(@item['updated']) if @item['updated']
-    h[:flipped] = Time.at(@item['flipped']) if @item['flipped']
+    # TODO: #75 consider other implementations of the null Time object that may
```

Comment:
@pdacostaporto if you format this according to PDD requirements it will picked up and converted to a new ticket, see: https://github.com/yegor256/pdd